### DISCUSSION TITLE
How to get assignees to a an issue created

### DISCUSSION THREAD

**Discussion by fretagi** [community/community#22139] [New to GitHub]
<p>I am not able to get any assignee to an issue created</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi fretagi! :wave:</p>
<p>Although you can create an issue on any repository you have read access to (that hasn’t disabled issues entirely) you need to have write access to be able to assign someone!</p>
<p><a href="https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users" class="onebox" target="_blank">https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users</a></p>
<p>I hope that explains what’s going on!</p>
